dashboard/app/views/home/index.html.haml

```diff
@@ -5,6 +5,10 @@
 
 - if current_user && current_user.teacher? && !current_user.accepted_latest_terms?
   = render partial: 'layouts/terms_interstitial'
+- if current_user && current_user.show_race_interstitial?
+  - current_user.race_interstitial_shown = true
```

Comment:
It turns out that for several reasons, I'm taking a different approach here, the primary one being that we actually don't care if the user *sees* the dialog more than once, we care if they have interacted with it. And if they have interacted with it, we'll have data stored on the server, so we'll block the rendering server-side instead of doing anything on the client.